adobe/aem-core-cif-components
# CIF-1711 - [React Components] Shipping Address form bugs

## Description

The missing label of the `Use Address` bug was fixed by updating the name of the prop that is used to pass the label.

The `region` error was fixed by adding a dummy empty entry in the regions dropdown to force a selection. Otherwise, the form didn't receive the selection event from the dropdown and the value was empty when the form was submitted.

A trivial update: I added the `graphqlMethod` field in the configuration object passed to the mock `ConfiContext`

## Related Issue






CIF-1711

## Motivation and Context


Bug fixing

## How Has This Been Tested?





Make sure you are using a Magento 2.4 instance!

1. Add a product to the cart
2. Start the checkout process and open the "Shipping Address" form
3. Check that the `Use Address` button has the correct label
4. Select a country that has regions, i.e. Romania or Germany
5. The "regions" dropdown will be rendered
- check that you cannot submit the form until you select a region
- check that you can select a region and the form is submitted successfully.

Re-open the form again and check that the country / region selection is the same as the one submitted.

## Screenshots (if appropriate):

## Types of changes




## Checklist:
